cmd/restic/lock_test.go

```diff
@@ -168,3 +170,47 @@ func TestLockSuccessfulRefresh(t *testing.T) {
 	// unlockRepo should not crash
 	unlockRepo(lock)
 }
+
+func TestLockWaitTimeout(t *testing.T) {
+	repo, cleanup, env := openTestRepo(t, nil)
+	defer cleanup()
+
+	elock, _, err := lockRepoExclusive(context.TODO(), repo, env.gopts)
+	test.OK(t, err)
+
+	wait := time.Second
```

Comment:
can we reduce that duration further? Our tests already take longer than I'd like.